### ISSUE TITLE
Unable to update Opset Version due to Batch Normalization

### ISSUE THREAD

**Issue by zjgarvey** [nod-ai/SHARK-ModelDev#859]
Removing the opset update, [these models](https://gist.github.com/zjgarvey/fd9526f4c0566d1eff65308419739516#file-import_list_5-txt) pass ORT inference, but also require basic ORT optimizations to clean up BatchNormalization Nodes with missing outputs.

Including an update soon to remove the opset version updating and apply basic ORT optimizations for these models. They should now report a PASS.

**Comment by zjgarvey** [comment]
partially resolved with <https://github.com/nod-ai/SHARK-TestSuite/pull/365>

There are a few models left commented out that require this change, but for some reason, fixing them seems to cause a test runner crash (OOM) for me locally on import phase. Since those models OOM on import anyway, we need to fix issue <https://github.com/nod-ai/SHARK-ModelDev/issues/861> before uncommenting the two remaining models which need the fix.
